saleor/graphql/product/tests/test_bulk_delete.py

```diff
@@ -507,6 +507,37 @@ def test_delete_products_trigger_webhook(
     mocked_recalculate_orders_task.assert_not_called()
 
 
+@patch("saleor.plugins.webhook.plugin.trigger_webhooks_for_event.delay")
```

Comment:
Patch manager method instead of WebhookPlugin. You need to confirm that we run the expected Manager method (like `product_deleted`). No need to enable or mock WebhookPlugin. 
`@patch("saleor.plugins.manager.PluginsManager.order_updated")`

Reply:
But I dont want path plugin manager method, I want to run it and mock only call to celery.